### PR TITLE
 Make import-memory tool take command line args

### PR DESCRIPTION
**Pull request by vgrichina**

### CODE REVIEW

**Comment by vgrichina** [comment]
@evgenykuzyakov probably should be rebased once you clean-up your changes.

File: core/wasm/runtest/generate-wasm/import-memory/src/main.rs [hunk 1]
```diff
@@ -42,15 +43,31 @@ fn externalize_mem(
     builder.build()
 }
 
+fn help() {
```
Comment (bowenwang1996) [inline core/wasm/runtest/generate-wasm/import-memory/src/main.rs]: Why do you need to make this a function

Comment (vgrichina) [reply core/wasm/runtest/generate-wasm/import-memory/src/main.rs]: Copy pasta, but actually I think makes sense to extract conversion code well.

**Comment by vgrichina** [comment]
@evgenykuzyakov done.

**Comment by evgenykuzyakov** [comment]
Please also modify generate-wasm/build.sh

**Pull request merged**